src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -94,6 +94,37 @@ class SourcesTree extends Component<Props, State> {
       debuggeeUrl,
       sources
     });
+    this.orderTree.bind(this);
+  }
+
+  orderTree() {
```

Comment:
minor nit, you can do `orderTree = () => {}` and avoid the bind calls

Reply:
thanks @jasonLaster